web/react/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "allowJs": false,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Why `commonjs` and not `esnext`?

Reply:
when imported in Kiali, with `esnext` it is working fine at runtime ... except the jest tests because it seems jest does not know how to read imports in the `esnext` format. So I had test failures in Kiali because of that.